code/src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { HappyThought } from 'components/HappyThought'
+import { HappyForm } from 'components/HappyForm';
+
+const url= "https://happy-thoughts-technigo.herokuapp.com/thoughts"
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState ([])
+  const [postedMessage, setPostedMessage] = useState ("")
+
+  useEffect(() => {
+    fetch(url)
+      .then(res => res.json())
+      .then(json => setThoughts(json))
+  }, [postedMessage])
+
+  const onFormSubmit = message => {
+    setPostedMessage(message)
+  }
+
+  const onLiked = thoughtId => {
+    console.log("Logging in the APP.js" , thoughtId)
```

Comment:
Before you’re done with the project you should delete all of the console.logs 🥳

Reply:
👍